packages/@react-spectrum/list/src/index.ts

```diff
@@ -1,2 +1,2 @@
-export {ListView} from './ListView';
+export * from './ListView';
```

Comment:
Does this have to be a `*`? IMO we should stick to the explicit exports, prevents any accidental exports

Reply:
So, this is failing locally but I pushed a PR just to have others take a look, and it passes here. Still investigating why it fails locally though. 

No specific reason to switch to `*` besides trying something else to see why the build was failing. I can definitely switch this back to implicit.